backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -179,12 +180,47 @@ export class ContributionResolver {
   async listAllContributions(
     @Args()
     { currentPage = 1, pageSize = 5, order = Order.DESC }: Paginated,
+    @Arg('filterState', () => [ContributionStatus], { nullable: true })
+    filterStates: ContributionStatus[] | null,
+    @Ctx() context: Context,
   ): Promise<ContributionListResult> {
+    const where: {
+      contributionStatus?: FindOperator<string> | null
+    } = {}
+    const typeStatus = Object.values(ContributionStatus)
+    const user = getUser(context)
+    i18n.setLocale(user.language)
+    if (filterStates !== null) {
+      const filterStateArray = []
+      const length = filterStates.length
+      let i = 0
+      for (i; i < length; i++) {
+        const filterState = filterStates[i]
+        if (!typeStatus.includes(filterState)) {
+          logger.error(
+            `${i18n.__('error.contributions.wrongFilterState', {
+              contributionState: filterStates[i],
+            })}`,
+          )
+          throw new Error(
+            `${i18n.__('error.contributions.wrongFilterState', {
+              contributionState: filterStates[i],
+            })}`,
+          )
```

Comment:
Open to discussion if we want it or not! Is only a test of mine to see if I understand the i18n api right!